specification/postgresql/resource-manager/readme.typescript.md

```diff
@@ -7,7 +7,34 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/arm-postgresql"
-  output-folder: "$(typescript-sdks-folder)/sdk/postgresql/arm-postgresql"
   clear-output-folder: true
   generate-metadata: true
 ```
+
+### TypeScript multi-api
```

Comment:
This multi-api is not required. Since we are not going to combine both. So, Any reason this is included?

Reply:
You are correct. Removed